src/api/types.ts

```diff
@@ -21,13 +21,13 @@ interface ScorecardCreator {
 
 export interface Scorecard {
   creator: ScorecardCreator;
+  description?: string;
```

Comment:
Should this be 
```suggestion
  description: string | null;
```
?

Reply:
I'll lead to o lot of cascade changes in multiple files/types so I would leave it as it is.